resume.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+  <html>
+    <head>
+      <title> Rossy </title>
+        	<link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400&display=swap" rel="stylesheet">
+    		<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+      		<style>
+                  body {
+    padding-left: 10%;
+    padding-right: 10%;
+    margin-bottom: 10%;
+    background-color: #c3fbf4;
+}
+.Sticky {
+    width: 20vw;
+    height: 30vh;
+    position: fixed;
+    top: 0px;
+    z-index: 0;
+
+}
+.Main {
+    margin-top: 45vh;
+    background-color: #FF6978;
+    width: 80vw;
+    height: relative;
+    position: relative;
+    bottom: 30%;
+    z-index: 1;
+    opacity: 80%;
+}
+text {
+    font-family: "Lato", sans-serif;
+}
+h1 {
+    font-family: "Lato";
+	font-size: 3.5em;
+	text-indent: 0px;
```

Comment:
This reiterates the [default browser styles](https://browserdefaultstyles.com/) and, therefore, can be removed:
```suggestion
```